burn-wgpu/src/kernel/matmul.rs

```diff
@@ -1,16 +1,49 @@
-use super::{build_info, KernelSettings};
+use std::cmp::{max, min};
+
+use super::{build_info, DynamicKernelSettings, SourceTemplate, StaticKernel};
 use crate::{context::WorkGroup, element::WgpuElement, kernel_wgsl, tensor::WgpuTensor};
 use burn_tensor::Shape;
 
-const BLOCK_SIZE: usize = 16;
+const B_M: usize = 128;
+const B_N: usize = 128;
+const B_K: usize = 8;
+const T_M: usize = 8;
+const T_N: usize = 8;
```

Comment:
Could we add comments for each constant to signify what B_M stands for, e.g.? The names to short to know what they're.